### PR TITLE
Plumb mobile os version to go side

### PR DESCRIPTION
**Pull request by zapu**
Rationale: we need to know Android version on go side, because secret store behavior will be different on old (android <=22) systems.

### CODE REVIEW

**Comment by zapu** [comment]
Android builds and seems to work, I couldn't test iOS though

**Comment by mmaxim** [comment]
I thought you had a Mac somewhere? If you do it would be great to test it first

**Comment by zapu** [comment]
I had some issues in building ios, but might as well figure it out now if I'm going to do more frontend work in near future. thanks!

**Comment by zapu** [comment]
Tried it with xcode, fixed few issues, seems to work at least on simulators / emulators - confirmed with logcat on android side and in xcode terminal that right version string is getting plumbed.

**Comment by zapu** [comment]
Rebased, @mmaxim i think this is ready for review

File: shared/android/app/src/main/java/io/keybase/ossifrage/util/MobileOSVersion.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package io.keybase.ossifrage.util;
+
+public final class MobileOSVersion {
+    public static String Get() {
+        return Integer.toString(android.os.Build.VERSION.SDK_INT);
```
Comment (buoyad) [inline shared/android/app/src/main/java/io/keybase/ossifrage/util/MobileOSVersion.java]: Can we get this without adding this util?

Comment (zapu) [reply shared/android/app/src/main/java/io/keybase/ossifrage/util/MobileOSVersion.java]: Yes - alternative is adding doing `Integer.toString(android.os.Build.VERSION.SDK_INT)` in both call sites of `initOnce`. So in `KeybasePushNotificationListenerService.java` and `MainActivity.java`. I can change it if that's preferable. Thanks!

**Pull request merged**